src/config.rs

```diff
@@ -620,8 +625,12 @@ pub struct DbConfig {
     pub defaultcf: DefaultCfConfig,
     pub writecf: WriteCfConfig,
     pub lockcf: LockCfConfig,
-    pub raftcf: RaftCfConfig,
     pub titan: TitanDBConfig,
+
+    // Deprecated since v3.x.
+    #[doc(hidden)]
+    #[serde(skip_serializing, skip_deserializing)]
```

Comment:
I don't think you should skip deserializing. What if they are customized before?